### PR TITLE
feat(threads): add `invitable`

### PR DESCRIPTION
**Pull request by advaith1**
* https://github.com/discord/discord-api-docs/pull/3600

### CODE REVIEW

File: rest/v9/channel.ts [hunk 1]
```diff
@@ -130,6 +130,12 @@ export interface RESTPatchAPIChannelJSONBody {
 	 * Channel types: text, news
 	 */
 	default_auto_archive_duration?: ThreadAutoArchiveDuration;
+	/**
+	 * Whether non-moderators can add other non-moderators to the thread
+	 *
+	 * Channel types: privateThread
```
Comment (vladfrangu) [inline rest/v9/channel.ts:136]: What's this?

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: rest/v9/channel.ts (hunk 1 above)
Comment (advaith1) [reply rest/v9/channel.ts:136]: ?

Comment (vladfrangu) [reply rest/v9/channel.ts:136]: What does it represent? Can you be more specific wkth the jsdoc, like `Only present on the following channel typez: {@link ChannelTypes.PrivateThread}` or similar

Comment (advaith1) [reply rest/v9/channel.ts:136]: well all the other fields use the same format, should I change all of them? that feels out of the scope of this PR

Comment (vladfrangu) [reply rest/v9/channel.ts:136]: Do they..? Ugh.. Yeah, I guess for consistency, leave it as is for now, but we gotta fix that

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**